src/MockBot.Api/Models/Constants.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System.Diagnostics.CodeAnalysis;
+
+namespace MockBot.Api.Models
+{
+    // No logic so no unit tests are required
+    [ExcludeFromCodeCoverage]
+    public static class Constants
+    {
+        public const string Unknown = "Unknown";
+        public const string XSendToDmr = "Dmr";
+        public const string XSendToClassifier = "Classifier";
+        public const string XModelType = "application/vnd.classifier.classification+json;version=1";
+        public const string ContentTypePlain = "text/plain";
+        public const string Empty = "";
```

Comment:
Yes, that is a valid question indeed!

The reason we use a constant rather than the built in `string.empty` is to avoid a Stryker mutation that populates the string with something other than empty. By having it in a constants file which is excluded, we avoid that mutation. Not sure if this is a valid approach or not, shall we discuss?

@joonasroosalung also raised the same concern.

Reply:
Look to add a test to validate the behaviour for when `string.empty` is not empty. This should address the Mutation and avoid the need to have and empty string as a constant.

Reply:
I have now removed this constant and refactored the code slightly to use optional params.